packages/toasty/test/specs/cta.buttons.feature.js

```diff
@@ -0,0 +1,33 @@
+import ButtonsSection from '../pageobjects/buttons.section';
+import { ctaButtonEnabled, ctaButtonDisabled, ctaButtonHover } from '../entities/buttons';
+import { expect as expectChai } from 'chai';
+
+
+describe('CTA Buttons', () => {
+    it('cta button enabled should has the expected CSS values', () => {
+
+        ButtonsSection.open();
+        let expectedValues = ctaButtonEnabled;
+        let actualValues = ButtonsSection.getCTAButtonEnabledAttributes();
+        expect(actualValues).toStrictEqual(expectedValues);
```

Comment:
I like this approach, tests just should be the comparison of a actual value against a expected value. Keep it simple sometimes is very hard to achieve.